integration-tests/src/test/scala/za/co/absa/spline/ExcelSpec.scala

```diff
@@ -63,7 +66,7 @@ class ExcelSpec extends AsyncFlatSpec
         } yield {
           plan1.operations.write.append shouldBe false
           plan1.operations.write.extra.get("destinationType") shouldBe Some("excel")
-          plan1.operations.write.outputSource shouldBe s"file:$filePath"
+          plan1.operations.write.outputSource shouldBe s"$fileUri"
```

Comment:
Redundant string interpolation:
```suggestion
          plan1.operations.write.outputSource shouldBe fileUri
```

Reply:
Agreed. Solved in following series of commit.